docs/guide/node_in_cloud.md

```diff
@@ -0,0 +1,152 @@
+2019-07-30 17:33:34.149030: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x52f7090 executing computations on platform Host. Devices:
+2019-07-30 17:33:34.149057: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
+
+Total params: 594922
+Trainable params: 594922
+Non-trainable params: 0
+_________________________________________________________________
+Epoch 1 / 20
+Epoch 1 / 20
+========================>----------------------------------------------------------------------------------: 35.5
+```
+
+3. Make sure you ignore build artifacts, such as node_modules, in your .gitignore file.
+
+### Create and deploy the heroku app
+
+1. Create a new app on heroku website
+2. Commit your change and push to heroku master
+
+```
+$ git init
+$ heroku git:remote -a your-app-name
+$ git add .
+$ git commit -m "First Commit"
+$ git push heroku master
+```
+
+3. In the build logs, you should be able to see the tfjs-node package downloading the TensorFlow C Library and loading TensorFlow Node.js native addon:
+
+```
+remote: -----> Installing dependencies
+remote:        Installing node modules (package.json)
+remote:
+remote:        > @tensorflow/tfjs-node@1.2.5 install /tmp/build_de800e169948787d84bcc2b9ccab23f0/node_modules/@tensorflow/tfjs-node
+remote:        > node scripts/install.js
+remote:
+remote:        CPU-linux-1.2.5.tar.gz
+remote:        * Downloading libtensorflow
+remote:
+remote:        * Building TensorFlow Node.js bindings
+remote:        added 92 packages from 91 contributors and audited 171 packages in 9.983s
+remote:        found 0 vulnerabilities
+remote:
+```
+
+In the process logs on heroku you should be able to see the model training logs:
+
+```
+Total params: 594922
+Trainable params: 594922
+Non-trainable params: 0
+_________________________________________________________________
+Epoch 1 / 20
+Epoch 1 / 20
+====>--------------------------------------------------------------------: 221.9
+```
+
+You can also start or debug the process in heroku [console](https://devcenter.heroku.com/articles/heroku-dashboard#application-overview).
+
+### Using tfjs-node prior to version 1.2.4
+
+If you are using tfjs-node package before version 1.2.4, the package requires g++ to compile the node native addon from source files. You will have to make sure your stack has Linux build-essential package (newer version stack may not have it on default).
+
+## Deploy Node.js project on Google Cloud Platform
+
+###Prerequisite:
+
+1. Have a valid Google Cloud Project with billing account
+2. Install Google Cloud [client tool](https://cloud.google.com/storage/docs/gsutil_install)
+3. Add app.yaml file to configure the [Node.js Runtime](https://cloud.google.com/appengine/docs/flexible/nodejs/runtime)
+
+### Deploy app to GCloud
+
+Run `gcloud app deploy` to deploy the local code and configurations to App Engine. In the deploy logs you should be able to see tfjs-node is installed:
```

Comment:
... "able to see that tfjs-node is installed" ... (add that).

Reply:
Done.